ablySpec/RealtimeClientChannel.swift

```diff
@@ -552,6 +552,60 @@ class RealtimeClientChannel: QuickSpec {
 
                 }
 
+                // RTL6g
+                context("Identified clients with clientId") {
+
+                    // RTL6g1
+                    context("When publishing a Message with clientId set to null") {
+
+                        // RTL6g1a
+                        pending("should be unnecessary to set clientId of the Message before publishing") {
+                            let options = AblyTests.commonAppSetup()
+                            options.autoConnect = false
+                            let client = ARTRealtime(options: options)
```

Comment:
Yes, but unless I'm reading the code very wrong, isn't that what @ricardopereira did initially and you asked him to change?

From the code snippet commented [here](8 days ago):

```swift
+                        pending("should be unnecessary to set clientId of the Message before publishing") {
+                            let options = AblyTests.commonAppSetup()
+                            options.autoConnect = false
+                            options.clientId = "client_string"
```

That's setting the clientId in the connection options, not in the message.